.buildkite/pipeline.yml

```diff
@@ -5,17 +5,17 @@ env:
   IMAGE_ID: xcode-15.4
 
 steps:
-  - label: "🕵️ Lint"
+  - label: ":sleuth_or_spy: Lint"
     command: make lint
     agents:
       queue: default
 
-  - label: "🔬 Test"
+  - label: ":microscope: Test"
     command: make test
     agents:
       queue: default
 
-  - label: "☢️ Danger - PR Check"
+  - label: ":radioactive_sign: Danger - PR Check"
```

Comment:
Such a change in steps labels will require updating the branch protection settings on GitHub with the new derived names for the required checks.

Given the long names Buildkite already ends up generating on the existing GitHub checks (e.g. `buildkite/ci-toolkit-buildkite-plugin/swift-standalone-swift-package-autodetect`) and how using `:emoji:` codes like here will also lead to long labels (like `buildkite/ci-toolkit-buildkite-plugin/radioactive-sign-danger-pr-check`)… I think this could be a good occasion to use this repo as guinea pig for paaHJt-78h-p2, as we could consider:

 - Turning off the `publish_commit_status` setting on the pipeline (the one that makes Buildkite generate status based on label names)
 - Instead declare `notify` attributes (with e.g. `{github_commit_status: { context: "Danger - PR Check" } }`) explicitly for every single step in the `pipeline.yml`

PS: Given all of this, you might want to revert this commit from the time being and instead apply those changes in a separate PR instead.